objects/src/transaction/tx_args.rs

```diff
@@ -66,6 +115,16 @@ impl TransactionArgs {
     pub fn get_note_args(&self, note_id: NoteId) -> Option<&Word> {
         self.note_args.get(&note_id)
     }
+
+    /// Returns a reference to the args [AdviceMap].
+    pub fn get_advice_map(&self) -> &AdviceMap {
+        &self.advice_map
+    }
+
+    /// Returns a mutable reference to the args [AdviceMap].
+    pub fn get_advice_map_mut(&mut self) -> &mut AdviceMap {
+        &mut self.advice_map
+    }
```

Comment:
nit: since we are just returning references, I'd rename these into `advice_map()` and `advice_map_mut()`.

Also question: why do we need to return the mutable reference? If we are not using it, I'd rather remove it.